src/main/java/ai/deepcode/javaclient/DeepCodeRestApi.java

```diff
@@ -31,29 +40,64 @@ private DeepCodeRestApi() {}
 
   private static final String API_URL = "https://www.deepcode.ai/";
 
-  private static Retrofit retrofit = buildRetrofit(API_URL);
+  private static Retrofit retrofit = buildRetrofit(API_URL, false);
 
   // Create simple REST adapter which points the baseUrl.
-  private static Retrofit buildRetrofit(String baseUrl) {
-    OkHttpClient client = new OkHttpClient.Builder()
+  private static Retrofit buildRetrofit(String baseUrl, boolean disableSslVerification) {
+    OkHttpClient.Builder builder = new OkHttpClient.Builder()
             .connectTimeout(100, TimeUnit.SECONDS)
             .writeTimeout(100, TimeUnit.SECONDS)
-            .readTimeout(100, TimeUnit.SECONDS).build();
+            .readTimeout(100, TimeUnit.SECONDS);
+    if (disableSslVerification) {
+      X509TrustManager x509TrustManager = buildUnsafeTrustManager();
+      final TrustManager[] trustAllCertificates = new TrustManager[]{ x509TrustManager };
+
+      try {
+        SSLContext sslContext = SSLContext.getInstance("SSL");
+        sslContext.init(null, trustAllCertificates, new SecureRandom());
+        SSLSocketFactory sslSocketFactory = sslContext.getSocketFactory();
+        builder.sslSocketFactory(sslSocketFactory, x509TrustManager);
+      } catch (NoSuchAlgorithmException | KeyManagementException e) {
+        //TODO(pavel): extract Retrofit and OkHttpClient into configuration object to simplify API client building.
+        e.printStackTrace();
```

Comment:
@pavel-snyk just curious if ssl configuration fails we still works, right? And we don't need this exception details at upper level?